Cargo.toml

```diff
@@ -63,6 +63,8 @@ members = [
   "wallet/wallet-node-client",          # Wallet-to-node communication tools.
   "wallet/wallet-address-generator",    # Wallet address generator binary.
   "wallet/wallet-address-generator-lib",# Wallet address generator lib.
+  "wallet/wallet-rpc-lib",              # Wallet RPC definitions library.
+  "wallet/wallet-rpcd",                 # Wallet RPC daemon binary.
```

Comment:
We don't use the suffix `d` usually. I consider it an outdated standard that was used a long time ago when file names were short by necessity. Please call it wallet-rpc-daemon, in symmetry with node-daemon.